x-pack/test/security_solution_endpoint/apps/endpoint/index.ts

```diff
@@ -22,7 +22,7 @@ export default function (providerContext: FtrProviderContext) {
       log.warning('These tests are being run with an external package registry');
     }
 
-    const registryUrl = getRegistryUrl() ?? DEFAULT_REGISTRY_URL;
+    const registryUrl = getRegistryUrl() ?? getRegistryUrlFromIngest();
```

Comment:
Just a naming comment: would it make sense to rename those functions to `getRegistryUrlFromConfig()` and `getRegistryUrlFromDefaults()` or something in that direction? That would make these lines immediately understandable, without having to dig what these different `getRegistry` functions do.

Reply:
I think that's a good idea. I paused for a moment to think about names (*docker, etc) and couldn't think of one.

Happy to update these to anything. LMK any opinions, @jonathan-buttner

Reply:
Ah good point, the name I used was too generic. @jfsiii if you have time and don't mind switch the name of the `'../../../security_solution_endpoint_api_int/registry'` to something like `getRegistryUrlFromTestEnv` that'd be great.

Otherwise I can do it in a follow up PR to this 👍

Reply:
@jonathan-buttner np. will do.